desktop-app/app/components/Header/index.js

```diff
@@ -53,22 +58,45 @@ const Header = () => {
         toastClassName={classes.darkToast}
       />
       <AppNotification />
+      <ZenButton
+        onClick={() => props.setHeaderVisibility(!props.isHeaderVisible)}
+      />
     </div>
   );
 };
 
 const useStyles = makeStyles(theme => ({
   container: {
+    position: 'relative',
     background: theme.palette.background.l1,
     display: 'flex',
     flexDirection: 'column',
     width: '100%',
-    padding: os.platform() === 'darwin' ? '20px 0 5px' : '0 0 0',
+    padding: os.platform() === 'darwin' ? '0 0 5px' : '0 0 0',
```

Comment:
Removing the padding is going to make the OS's minimize/maximize buttons overlap on the toolbar right?

Reply:
Nevermind, just noticed the `Toolbar` component.